pages/common/todo.sh.md

```diff
@@ -0,0 +1,28 @@
+# todo.sh
+
+> Simple and extensible shell script for managing your `todo.txt` file.
+> More information: <https://github.com/todotxt/todo.txt-cli>.
+
+- List every item:
+
+`todo.sh ls`
+
+- Add an item with project and context tags:
+
+`todo.sh add '{{description}} +{{project}} @{{context}}'`
+
+- Mark an item as [do]ne:
+
+`todo.sh do {{item_no}}`
+
+- Remove an item:
+
+`todo.sh rm {{item_no}}`
+
+- Set an item's [pri]ority (A-Z):
+
+`todo.sh pri {{item_no}} {{priority}}`
+
+- Replace an item:
+
+`todo.sh replace {{item_no}} '{{new_description}}'`
```

Comment:
```suggestion
`todo.sh replace {{item_no}} {{new_description}}`
```

Reply:
@acuteenvy the todo.sh documentation encourage using strings for the descriptions. Why do you think I should remove it ?

Reply:
The placeholder contains no spaces, that's why I thought about removing quotes. But there's a lot of options that can be put inside the string (`@context`, etc.), so never mind.